### PR TITLE
Upgrade raster images with rebranded theme

### PR DESCRIPTION
**Pull request by lparisc**
Updating images used/generated in:
- protocol documents
- tic-tac-toe tutorial

### CODE REVIEW

File: docs/60_protocol-guides/30_network_peer_protocol.md [hunk 1]
```diff
@@ -533,6 +542,8 @@ Therefore, the node’s LIB block is updated first, followed by the node’s hea
 
 Case 1 above, where the node’s LIB block needs to catch up with the peer’s LIB block, is depicted in the below diagram, before and after the sync (Note: inapplicable branches have been removed for clarity):
```
Comment (bobgt) [inline docs/60_protocol-guides/30_network_peer_protocol.md]: Does the "Case 1" refer to step 1 in 3.3 Catch-Up Mode? If yes, maybe we can call them as "cases" or "steps." I had to search above where is Case 1. Upon reading I understood that step 1 is referred to as "Case 1."

File: docs/60_protocol-guides/30_network_peer_protocol.md [hunk 2]
```diff
@@ -599,6 +611,8 @@ In the above diagram, the node’s local chain syncs up with the peer’s local
 
 After the node’s LIB block is synced with the peer’s, there will be new blocks pushed to either chain. Case 2 above covers the case where the peer’s chain is longer than the node’s chain. This is depicted in the following diagram, which shows the node and the peer’s local chains before and after the sync:
```
Comment (bobgt) [inline docs/60_protocol-guides/30_network_peer_protocol.md]: Same comment about the "case."

**Review by bobgt — APPROVED** [review]
The pages look great with these diagrams! Awesome!

File: docs/60_protocol-guides/30_network_peer_protocol.md (hunk 1 above)
Comment (lparisc) [reply docs/60_protocol-guides/30_network_peer_protocol.md]: Good catch. Should be "Steps". I will update.

**Pull request merged**